src/Models/Model.php

```diff
@@ -129,7 +130,11 @@ public function getId()
      */
     public function getKeyName()
     {
-        return strtolower($this->getName() . '_id');
+        if (method_exists($this->module, 'getApiKeyName')) {
+            return $this->module->getApiKeyName();
+        } else {
```

Comment:
can you use early returns here?
```php
if (method_exists($this->module, 'getApiKeyName')) {
    return $this->module->getApiKeyName();
}

return strtolower($this->getName() . '_id');
```

or even better: can you declare that method in the Model class with the standard implementation, and override it in the child classes that need it? Thanks!